### PR TITLE
fix conftest.py

### PR DESCRIPTION
**Pull request by rgirl0801**

### CODE REVIEW

File: tests/test_inventory.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import allure
+import pytest
+
+from constants import SOTRING_AZ, SOTRING_ZA, SOTRING_PRICE_ASC, SOTRING_PRICE_DESC
```
Comment (akioikeda) [inline tests/test_inventory.py:4]: What is constants? How to get it? Should I install it somehow?

Comment (rgirl0801) [reply tests/test_inventory.py:4]: No, I think, you should delete them.

**Review by akioikeda — CHANGES_REQUESTED** [review]
- In my case the test is failing here:
FAILED tests/test_inventory.py::TestCartClass::test_change_add_button_text - selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id="add-to-cart-sauce-labs-bike-light"]
Please fix
- fix import

File: tests/test_inventory.py (hunk 1 above)
Comment (rgirl0801) [reply tests/test_inventory.py]: Done

**Comment by rgirl0801** [comment]
> * In my case the test is failing here:
>   FAILED tests/test_inventory.py::TestCartClass::test_change_add_button_text - selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id="add-to-cart-sauce-labs-bike-light"]
>   Please fix
> * fix import

Sounds weird. 
Because my test failed by reason of:
FAILED tests/test_inventory.py::TestCartClass::test_change_add_button_text - selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id="user-name"]
However, it works in another case(test_change_bage_when_adding e.g.)

**Comment by akioikeda** [comment]
> > * In my case the test is failing here:
> >   FAILED tests/test_inventory.py::TestCartClass::test_change_add_button_text - selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id="add-to-cart-sauce-labs-bike-light"]
> >   Please fix
> > * fix import
> 
> Sounds weird. Because my test failed by reason of: FAILED tests/test_inventory.py::TestCartClass::test_change_add_button_text - selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: [id="user-name"] However, it works in another case(test_change_bage_when_adding e.g.)

Could you please run tests again on your system?

**Pull request closed without merging**